client/components/quizcreator/QuizName.jsx

```diff
@@ -5,7 +5,12 @@ const QuizName = ({ name, handleNameChange }) => {
   return (
     <div>
       <form>
-        <TextField id="quiz-name" value={name} onChange={(e) => handleNameChange(e.target.value)} label="Quiz Name" />
+        <TextField
+          id="quiz-name"
+          value={name}
+          onChange={(e) => handleNameChange(e.target.value)}
```

Comment:
{}?

Reply:
Leaving it the way it is, Ive created infinite render loops before and do not want to mess anything up

Reply:
I think that one was an accident. Whoops!